src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -270,27 +301,32 @@ public static void MapInvalidateMeasure(IViewHandler handler, IView view, object
 			(handler.PlatformView as PlatformView)?.InvalidateMeasure(view);
 		}
 
-		public static void MapContainerView(IViewHandler handler, IView view)
+		public static void MapNeedsContainer(IViewHandler handler, IView view, object? args)
 		{
-			if (handler is ViewHandler viewHandler)
-				handler.HasContainer = viewHandler.NeedsContainer;
-			else
-				handler.HasContainer = view.NeedsContainer();
+			if (handler is ViewHandler viewHandler && args is string mapper)
+				viewHandler.ChangesContainer.Add(mapper);
 
-			UpdateInputTransparentOnContainerView(handler, view);
+			handler.UpdateValue(nameof(INeedsContainerViewHandler.NeedsContainer));
 		}
```

Comment:
Then the NeedsContainer command adds the incoming property to the list to update later. After that, it calls the NeedsContainer PROPERTY mapper.